src/blockchains/ethereum/converter_version.ts

```diff
@@ -32,7 +32,7 @@ export async function get(web3, address) {
         const version = await rpc(contract.methods.version());
         const value = parse(type, version);
         if (value)
-            return {type: type, value: value};
+            return {type: type, value: String(Number(value.split('.').join('')))};
```

Comment:
any reason why not just replace '.' with '' here?

Reply:
`String.prototype.replace` replaces only the first occurrence of the input.
In addition to that, the input represents a regex and not a plain string.
In order to replace all occurrences, you need to use "regex syntax", which is why I always go for `split`/`join` instead.

Reply:
done

Reply:
Of course, but I can also use `value.split('.').join('')`; this is really minor...
BTW, I'm on node 12.20.0.